packages/@okta/vuepress-site/docs/concepts/identity-providers/index.md

```diff
@@ -0,0 +1,74 @@
+
+* All users are stored in Okta (a single user store). You can capture the profile attributes from an Identity Provider user and store those attributes in Okta's Universal Directory.
+
+* If a user updates their profile at the Identity Provider, those changes can be reflected inside Okta the next time that they use the provider to sign in. This is called profile sync.
+
+* Users can use multiple Identity Providers to sign in, and Okta links those profiles to a single Okta user, which is called account linking.
+
+* Okta supports group sync between Okta and enterprise SAML Identity Providers. This allows you to specify the groups that users should be added to. 
+
+## How Okta connects to external Identity Providers
+Okta sits between your application and the external provider. Your application only needs to talk to Okta, and Okta does the rest.
+
+### Sign-in process
+The sign-in process starts at the `/authorize` endpoint, and then goes out to the provider and back:
+
+1. In your application, the user clicks a button similar to: **Sign in with (Identity Provider)**.
+2. Your application redirects the browser to Okta.
+3. Okta redirects the browser to the Identity Provider.
+4. The user is prompted to sign in at the Identity Provider (if they aren't already) and to accept the permissions required by your app.
+5. The Identity Provider redirects the browser back to Okta.
+6. Okta processes the sign-in request and adds the user to your Okta organization's Universal Directory.
+7. Okta redirects the browser back to your application, just like any other sign-in request.
+
+![Social Login Flow width:](/img/social_login_flow.png "Social Login Flow width:")
+
+## Account Linking and Just-in-Time Provisioning
+When you allow your users to sign in to your app using their choice of Identity Providers, you can use Account Linking to help create a unified view of your users within your org. Additionally, you can use Just in Time (JIT) provisioning to create a seamless experience for users that sign-in to your application for the first time using their credentials from another Identity Provider.
+
+Users can use multiple Identity Providers to sign in, and Okta can link all of those profiles to a single Okta user. This is called Account Linking. If, for example, a user signs in to your app using a different Identity Provider than they used for registration, Account Linking can establish that the user owns both identities, allowing the user to sign in from either account.
+
+If a user signs in to your application for the first time using another Identity Provider, you can implement Just in Time (JIT) provisioning to automatically create an Okta account for them. JIT account creation and activation only works for end users who aren't already Okta users. (JIT does update the accounts of existing users during full imports.)
```

Comment:
That parenthetical sentence at the end is a bit unclear to me. Also, having a standalone parenthetical at the end of the paragraph seems...wrong.